debug/map/darkmode.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Leaflet debug page</title>
+
+	<link rel="stylesheet" href="../../dist/leaflet.css" />
+
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+	<link rel="stylesheet" href="../css/screen.css" />
+
+	<script src="../dist/leaflet-src.js"></script>
+
+	<style>
+		html, body{
+			background: #333333;
+		}
+

```

Comment:
Should this example have a fixed dark background?

I think authors are more likely to use either of the following methods:

1. Change background based on preference:

    ```css
    @media (prefers-color-scheme: dark) {
      :root {
        background-color: #333333;
      }
    }
    ```

2. Indicate to UAs it is compatible with dark mode and allow the UA to set the background:

    - using HTML:

      ```html
      <meta name="color-scheme" content="dark light">
      ```
    -  alternatively in CSS:

       ```css
       :root {
         color-scheme: dark light;
       }
       ```

References:

- https://html.spec.whatwg.org/multipage/semantics.html#meta-color-scheme
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name#:~:text=color-scheme
- https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme
- https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
- https://web.dev/color-scheme/
- https://web.dev/prefers-color-scheme/

Reply:
Good catch! I will change

Reply:
Just noting that the _commit message_ of https://github.com/Leaflet/Leaflet/pull/7612/commits/f2f7868d07dd734b8b50cc249090fc956c755005 indicates option 2 was used, though option 1 was used. 😋

Reply:
Option 2, option 1? I don't get it 😟

Reply:
@Falke-Design I was referring to https://github.com/Leaflet/Leaflet/pull/7612#discussion_r739652955.

> 1. Change background based on preference

> 2. Indicate to UAs it is compatible with dark mode and allow the UA to set the background

The commit description is "Indicate to UAs that the example htmls are compatible with dark mode", but that's not what https://github.com/Leaflet/Leaflet/commit/f2f7868d07dd734b8b50cc249090fc956c755005 does, it changes the background based on preferences. It's not a big deal, just a mismatch in the commit description.

Reply:
But I added in the `screen.css` following:
```
:root {
	color-scheme: light dark;
	}
```

Reply:
Ah, I somehow missed that. Sorry for all the fuss.